admin/config.yml

```diff
@@ -0,0 +1,19 @@
+backend:
+  name: github
+  repo: 18F/federalist-uswds-jekyll
+  branch: add_netlify_cms # Branch to update (optional; defaults to master)
```

Comment:
I would remove the `branch` configuration in favor of using the "editorial workflow" since that what gives us the nice draft and publish workflow.

Reply:
Sounds good.